fossunited/public/js/website.bundle.js

```diff
@@ -9,3 +9,7 @@ import "../../../../frappe/frappe/public/js/frappe/db.js";
 import "../../../../frappe/frappe/public/js/frappe/model/meta.js";
 import "../../../../frappe/frappe/public/js/frappe/form/controls/dynamic_link.js";
 import "../../../../frappe/frappe/public/js/frappe/utils/utils.js";
+import * as  PetiteVue from "petite-vue";
+window.PetiteVue = PetiteVue;
+import { autocomp } from "@knadh/autocomp";
```

Comment:
Why PetiteVue @RiyaMathew-11 ?

Reply:
Have used it to build the search and must-attend filter parts of the [event timeline](https://arc.net/l/quote/mlqvatab) - it's more of like a temporary measure to bring in these interactive components. 

We could rewrite the page in Vue maybe later. I used PetiteVue for a quick implementation.